src/main/java/seedu/duke/command/grade/GradeCommand.java

```diff
@@ -83,9 +85,9 @@ public CommandResult execute() {
             grade();
             return new CommandResult(MESSAGE_GRADE_MODULE_SUCCESS);
         } catch (InvalidGradeException e) {
-            return new CommandResult(MESSAGE_INVALID_GRADE);
+            return new CommandResult(centerString(MAX_WIDTH, EXCEPTION_HEADER) + "\n" + MESSAGE_INVALID_GRADE);
```

Comment:
Nice that you're using `centerString`, but generally all `CommandResult` messages don't use `centerString` I think, except for the error messages. Possible to remove to standardise?

Reply:
but the locations where i have used centerstrings are actually the error messages as the catches are in execute and they return commandresult instead of incorrectcommand thus for such specific cases i have included the error headers